src/cargo.ts

```diff
@@ -490,7 +490,11 @@ export function toDebianVersion(version: string, revision?: number): string {
   // Check if version is semver or cmake version
   if (version.includes("-")) {
     // HACK(fuzzypixelz): This is an oversimplification of the Debian Policy
-    debVersion = `${version.replace("-", "~")}-${revision ?? 1}`;
+    if (version.includes("alpha") || version.includes("beta") || version.includes("rc")) {
+      debVersion = `${version.replace("-", "~").replaceAll("-", ".")}-${revision ?? 1}`;
```

Comment:
```suggestion
      debVersion = `${version.replace("-", "~")}-${revision ?? 1}`;
```

Usually we just use `X.Y.Z-(rc|alpha|beta).N` as a pre-release.

Reply:
Indeed. In this case the `replaceAll` will have no impact as no more '-' will be remaining after the first `replace`. I added the `replaceAll` as a security to remove any eventual remaining '-' if anything changes in the future.